### DISCUSSION TITLE
Multi store on 1 portal instance

### DISCUSSION THREAD

**Discussion by thienvc** [openstore-ecommerce/OpenStore#181] [General]
Is it possible to create multiple Stores on the same portal?

**Comment by DNNMonster** [comment]
There may be different interpretations of multiple stores on the same portal...   But I think the short answer is no.

I'm familiar with an implementation that enables B2B and B2C from multiple sellers belonging to different types of roles all under one portal.  This means each seller has their own Back Office experience but not their own store...    This involved a lot of coding.

Others may have some better input but I think out of the box OS supports multiple store's sharing product and order data but technically each store is still it's own portal.  

I am not currently using those sharing features so Dave or others can correct me if I'm off base.

**Reply by thienvc** [reply]
Can I use OpenStore to use as sales management software for stores?

**Comment by DNNMonster** [comment]
I'm not sure what all your requirements are.

I think you may have to first get all your technical and functional specifications well defined.

At that point the community members here can probably give you some more specific answers about particulars.

But in my opinion, yes you can use Open Store for a sales management solution.

It's flexible, mature & the price is hard to beat.

I encourage you to spend some time inside the system so you can see what OS can and can't do for you.
